devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/maven/codestart.yml

```diff
@@ -2,16 +2,19 @@
 name: maven
 type: buildtool
 fallback: true
+output-strategy:
+  "mvnw": "executable"
+  "mvnw.cmd": "executable"
 language:
   base:
     data:
-      maven:
-        version:
-          kotlin: 1.3.72
-          scala: 2.12.8
-          scala-maven-plugin: 4.1.1
-          maven-compiler-plugin: 3.8.1
-          maven-surefire-plugin: 2.22.1
+      version:
```

Comment:
that explains the different naming strategy - okey; but then shouldn't the quarkus plugin version be here too ?

Reply:
@maxandersen the reason behind having `version.name` is that using yaml defining multiple `name.version` isn't really elegant:
```yaml
name1:
  version: 1.0
name2:
  version: 3.0
...
```

The difference is that here we only define versions, for quarkus or platform we also define artifact-id and group-id.

Maybe the `name.version` even if less elegant would make it more consistant?

Reply:
Ok I made the change in a new commit